src/client/app/offers/edit_offer.html

```diff
@@ -0,0 +1,160 @@
+<section>
+  <div class="container">
+      <div class="col-md-offset-1">
+          <h2 class="text-left">Нова пропозиція</h2>
+      </div>
+      <form class="form-horizontal">
+          <div class="form-group row">
+              <label class="control-label col-sm-2" for="title" >Назва</label>
+              <div class="col-sm-6">
+                  <!-- here should be pre-filled data from previous page -->
+                  <input type="text" class="form-control" name="heading" id="title" placeholder="Светр для дитячого будинку" ng-model="vm.editableOfferTittle" ng-show="vm.editorEnabled">
+                  {{vm.offerTittle}}
```

Comment:
styling?

this will not display anything, ng-model is bound to vm.editableOfferTittle, and you're displaying vm.offerTittle